tangelo/tangelo/__main__.py

```diff
@@ -194,6 +189,19 @@ def main():
         print tangelo_version
         return 0
 
+    # Make all logging go to stdout instead of stderr.
+    #
+    # The "cherrypy.error" logger contains two handlers by default - a
+    # NullHandler, and a StreamHandler that writes to stderr.  This piece of
+    # code simply removes the StreamHandler, modifies it to write to stdout, and
+    # then replaces it.
+    import logging
+    logger = logging.getLogger("cherrypy.error")
+    handler = logger.handlers[1]
+    logger.removeHandler(handler)
+    handler.stream = sys.stdout
+    logger.addHandler(handler)
+
```

Comment:
Could testing (and system services) take care of monitoring both streams? Seems a bit heavy-handed to merge stdout and stderr just to aid testing, when separating the streams can potentially be useful. When running a system service the following would log everything in one place: `tangelo > tangelo.log 2>&1`.